tests/add_subdirectory/vk_format_utils.c

```diff
@@ -0,0 +1,8 @@
+// Copyright 2023 The Khronos Group Inc.
+// Copyright 2023 Valve Corporation
+// Copyright 2023 LunarG, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+#include <vulkan/utility/vk_format_utils.h>
+
+void foo() {}
```

Comment:
what is this for?

Reply:
This tests whether the code is actually valid C code - since I just need to include it for the compiler to check, I should probably rename the function and add some usage just for testings sake.

Reply:
Is it possible to make this file "ephemeral", by generating only during the build phase? If that makes sense in this context. I guess this method is used by some configuration tools that generate small files to check for support of particular platform/language feature.

Reply:
It not only tests the C code is valid.

It helps test add_subdirectory / find_package utilization.

Reply:
IE it's integration testing to make sure we are writing good CMake for users to absorb our libaries